index/writer.go

```diff
@@ -333,7 +335,7 @@ func (w *indexWriter) write(rec *zed.Value) error {
 		if err != nil {
 			return err
 		}
-		w.frameKey = key
+		w.frameKey = key.Copy()
```

Comment:
Maybe use CopyFrom here?
```suggestion
		w.frameKey.CopyFrom(key)
```

Reply:
This doesn't work in general because the memory for the Value struct can get clobbered too.  Also, I think we'll soon end up making CopyFrom() go away.